script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -1850,15 +1985,17 @@ There―――
 [sha:5a1e51f081ee1493b101b9c94ae8899d24675dee]{
 -- Page 511, Offset 34752.
 -- 「なにそれ、美談のつもり？
-"What is this supposed to be, some sort of sob story?
+"What are you gonna do, tell me some sort of sob story?
 }
 [sha:8a74db369e0831e1ab74312b368d882389ee86c8]{
 -- Page 511, Offset 34753.
 -- そんな身勝手、許されるハズないじゃない」
-You don't truly believe that you have the right to do sometthing that selfish, do you?"
+Do you seriously believe you have the right to do something that selfish?"
 }
 [sha:8356df78d40d20e59c7c835a228c58e9a7e20e44]{
 -- Page 511, Offset 34754.
 -- 体を禍々しい<赤|いろ>に染めた、吸血鬼の姿があった。
-stands a vampire dyed in a sinister red.
+// Buddy that is 100% purple I refuse to be gaslit
+―――Stands the figure of a vampire,
+ their body draped head to toe in sinister purple.
```

Comment:
I think we were talking about her hair rather than the clothes, which are more of a mix, but both tones are closer to purple than her hair - 
![2023-02-26-155746_141x111_scrot](https://user-images.githubusercontent.com/2476799/221436892-34a36575-f0a0-4bd0-bcab-a677bda0cd11.png)

Reply:
I just kinda want a color that's like "red-purple" so it can be a BIT closer to the text.

Fuschia? Burgundy?

Reply:
Yeah I think fuchsia came up before. Works for me, it even perfectly matches _both_ colours on her clothes